apps/test/unit/util/curriculumRecommenderTest.js

```diff
@@ -181,29 +181,24 @@ describe('testRecommender', () => {
     const recommendedCurricula = getSimilarRecommendations(
-      SIMILAR_RECOMMENDER_TEST_COURSES,
+      FULL_TEST_COURSES,
       'month',
       'markInit1',
-      'subject2',
-      `topic1,topic2,${IMPORTANT_TOPICS[0]}`
+      'science',
+      `art_and_design,app_design,${IMPORTANT_TOPICS[0]}`
     ).map(curr => curr.key);
 
     expect(recommendedCurricula).to.deep.equal([
-      'multipleTopicsCourse', // 4 points = 2 overlapping topics * overlappingDesiredTopic(2)
-      'publishedWithinOneYearAgoCourse', // 2 points = publishedWithinOneYearAgo(2) [sorted highest of the 2-point courses by recent published_date]
-      'firstImportantTopicCourse', // 2 points = 1 overlapping topic * overlappingDesiredTopic(2)
-      'oneTopicCourse', // 2 points = 1 overlapping topic * overlappingDesiredTopic(2)
-      'multipleSchoolSubjectsCourse', // 2 points = 1 overlapping subject * overlappingDesiredSchoolSubject(2)
-      'marketingInitCourse1', // 2 points = hasDesiredMarketingInitiative(2)
-      'monthDurationCourse', // 2 points = hasDesiredDuration(2)
-      'publishedWithinTwoYearsAgoCourse', // 1 point = publishedWithinTwoYearsAgo(1) [sorted highest of the 1-point courses by recent published_date]
-      'secondImportantTopicCourse', // 1 point = hasImportantButNotDesiredTopic(1)
-      'oneSchoolSubjectCourse', // 1 point = hasAnySchoolSubject(1) [only receives points from hasAnySchoolSubject if numOverlappingDesiredSchoolSubjects == 0]
-      'featuredCourse', // 0 points [sorted highest of 0-point courses because it's marked as featured]
-      'emptyCourse', // 0 points
-      'nullCourse', // 0 points
-      'marketingInitCourse2', // 0 points
-      'weekDurationCourse', // 0 points
+      'fullTestCourse2' /* 10 points = hasDesiredDuration(2) + hasDesiredMarketingInitiative(2) + hasAnySchoolSubject(1) +
+                          (2 overlapping topics * overlappingDesiredTopic(2)) + publishedWithinTwoYearsAgo(1) */,
+      'fullTestCourse1' /* 9 points = hasDesiredMarketingInitiative(2) + hasAnySchoolSubject(1) + (2 overlapping topics * overlappingDesiredTopic(2))
+                          + publishedWithinOneYearAgo(2) */,
+      'fullTestCourse5' /* 6 points = hasDesiredDuration(2) + (1 overlapping subject * overlappingDesiredSchoolSubject(2)) +
+                          (1 overlapping topic * overlappingDesiredTopic(2)) [sorted highest of 6-point courses because it is featured] */,
+      'fullTestCourse3' /* 6 points = hasDesiredDuration(2) + (1 overlapping topic * overlappingDesiredTopic(2)) + hasAnySchoolSubject(1) +
+                          publishedWithinTwoYearsAgo(1) [sorted next-highest of 6-point courses by recent published_date] */,
+      'fullTestCourse4' /* 6 points = hasDesiredDuration(2) + (1 overlapping subject * overlappingDesiredSchoolSubject(2)) +
+                          (1 overlapping topic * overlappingDesiredTopic(2)) */,
```

Comment:
Since the Curriculum Catalog requires a lot of non-null fields (and many of the previous test curricula had only 1-2 fields set for the purposes of testing individual scoring questions), I had to add more test curricula that had all their fields set to be able to use the same test set here and in the end-to-end test in `CurriculumCatalogTest.jsx`.